admin/pr_as_branch

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+set -e
+set -u
+
+
+# Usage
+
+usage() {
+  program=$(basename "$0");
+
+  if [ $# != 0 ]; then echo "$@"; echo ""; fi;
+
+  echo "${program}: usage:";
```

Comment:
A more thorough usage message explaining the purpose of this script would be helpful.  Just a sentence or two, maybe even just a link to https://twistedmatrix.com/pipermail/twisted-python/2016-December/030921.html